docs/src/frontend/index.md

```diff
@@ -384,11 +384,20 @@ stdlib.fundFromFaucet(account, balance) => Promise<void>
 ```
 
 Adds the given balance of network tokens to a Reach account abstraction.
-This can only be used in private testing scenarios,
-as it uses a private faucet to issue network tokens,
-as well as certain public TestNet scenarios.
+This can only be used in private testing scenarios, as it uses a private faucet to issue network tokens, as well as certain public TestNet scenarios.
```

Comment:
This can only be used in private testing scenarios, because it uses a private faucet to issue network tokens. `{!js} fundFromFaucet` may also fund from certain public TestNet scenarios. 

What do you think of this alteration? The current format is a bit long and requires a couple read-throughs. 

I wonder if we have any idea what "Certain public TestNet scenarios" means. Be good to inquire in #help and see if we can get insight to add clarity to this statement.